content/en/docs/admin/admission-controllers.md

```diff
@@ -476,19 +465,17 @@ This admission controller has the following behavior:
   1. Evaluate the pod's node selector against the namespace-specific whitelist defined the plugin configuration file.
      Conflicts result in rejection.
 
-{{< note >}}
 **Note:** PodNodeSelector allows forcing pods to run on specifically labeled nodes. Also see the PodTolerationRestriction 
 admission plugin, which allows preventing pods from running on specifically tainted nodes.
-{{< /note >}}
+{: .note}
```

Comment:
Also revert to use Hugo shortcodes here and any other admonitions changed in this PR.